diagnostic.md

```diff
@@ -5,19 +5,19 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to handle requests from the front end such as INDEX, SHOW, CREATE, UPDATE, and DELETE.
```

Comment:
It does this by persisting our data and storing it between sessions. It also allows multiple users on different devices to interact over the same app.